infra/feast-operator/internal/controller/services/services.go

```diff
@@ -424,6 +425,33 @@ func (feast *FeastServices) setContainer(containers *[]corev1.Container, feastTy
 	}
 }
 
+func (feast *FeastServices) getVolumeMounts(feastType FeastServiceType) (volumeMounts []corev1.VolumeMount) {
+	appliedServices := feast.Handler.FeatureStore.Status.Applied.Services
+	if appliedServices == nil {
+		return []corev1.VolumeMount{} // Return an empty slice to avoid nil issues
+	}
+
+	switch feastType {
+	case OfflineFeastType:
+		if feast.isOfflinStore() {
+			return appliedServices.OfflineStore.VolumeMounts
+		}
+	case OnlineFeastType:
+		if feast.isOnlinStore() {
+			return appliedServices.OnlineStore.VolumeMounts
+		}
+	case RegistryFeastType:
+		if feast.isLocalRegistry() {
+			return appliedServices.Registry.Local.VolumeMounts
+		}
+	case UIFeastType:
+		if feast.isUI() {
+			return appliedServices.UI.VolumeMounts
+		}
+	}
+	return []corev1.VolumeMount{} // Default empty slice
+}
+
```

Comment:
your unit tests failures would likely be resolved if you changed this method to reflect `getLogLevel` in this same file.

Reply:
those are fixed after changing from to `appliedServices.OfflineStore.VolumeMounts` to `OfflineStore.Server.VolumeMounts`